options/customize.js

```diff
@@ -20,23 +22,50 @@ $(function() {
         "/filters";
 
     var jqxhr = $.ajax({
-        type: 'get',
+        type: "GET",
         url: syntaxURL,
-        success: function(e) {
+        success: function() {
             //since the ABP site uses a custom server side 404 handler, instead of returing us a 404 http status code
             //we need to parse the response text looking for a 404 message for the user.
```

Comment:
Ther server does now return a 404 code along with the HTML. Is this still needed?